sources/data-node/api/v2/trading_data.proto

```diff
@@ -358,6 +363,30 @@ message GetCandleDataResponse {
     repeated Candle candles = 1;
 }
 
+// -- Votes --
+// Request for Votes
+message GetVotesRequest {
```

Comment:
This doesn't look right, should the fields not just be `party_id`?

Reply:
Well, when you look at data-node code you see that you need to specify which direction you want the votes you requested.

Otherwise we can simply use two gaphql endpoints and in this case simply have the party